frontend/src/features/mapSideBar/SidebarContents/subforms/ClassificationForm.tsx

```diff
@@ -0,0 +1,124 @@
+  margin-left: 152px;
+  font-size: 12px;
+  width: 250px;
+  text-align: left;
+`;
+
+/** used to conditionally render the information display box with the desired content */
+const InfoBoxWithContent = (content: string) => {
+  return (
+    <InfoBox>
+      <p style={{ fontSize: '11px' }}>{content}</p>
+    </InfoBox>
+  );
+};
+
+interface IClassificationFormProps {
+  /** title to be displayed at the top of the form */
+  title?: string;
+  /** the classification set to be used with the component, could vary by form */
+  classifications: SelectOptions;
+  /** the field label for the select options (eg) Building Classification/Land Classification */
+  fieldLabel?: string;
+  /** pass down the parents nameSpace in order to access the appropriate fields */
+  nameSpace?: string;
+  /** pass the index to access the appropriate property if it is in a list format */
+  index?: string;
+  /** pass a tooltip for the classification field */
+  toolTip?: string;
+}
+
+export const ClassificationForm: React.FC<IClassificationFormProps> = ({
+  title,
+  classifications,
+  fieldLabel,
+  nameSpace,
+  index,
+  toolTip,
+}) => {
+  const formikProps = useFormikContext();
+
+  /** classId based on current formik values to determine which classsification information box to display */
+  let classId = index
+    ? (formikProps.values as any).data.buildings[index].classificationId
+    : (formikProps.values as any).data.classificationId;
+
+  const withNameSpace: Function = React.useCallback(
+    (name?: string) => {
+      return [nameSpace ?? '', name].filter(x => x).join('.');
+    },
+    [nameSpace],
+  );
+
+  const renderInfo = () => {
+    switch (classId) {
+      case CLASSIFICATIONS.CoreOperational:
+        return InfoBoxWithContent('Core Operational');
+      case CLASSIFICATIONS.CoreStrategic:
+        return InfoBoxWithContent('Core Strategic');
+      case CLASSIFICATIONS.SurplusActive:
+        return InfoBoxWithContent('Surplus Actice');
+      case CLASSIFICATIONS.SurplusEncumbered:
+        return InfoBoxWithContent('Surplus Encumbered');
+    }
+  };
+
+  return (
+    <>
+      <Row>
+        <Title>{title}</Title>
+      </Row>
+      <Row>
+        <Col md={6}>
+          <Row style={{ display: 'flex' }}>
+            <Label required>{fieldLabel}</Label>
+            <FastSelect
+              formikProps={formikProps}
+              type="number"
+              style={{ marginTop: '5px', display: 'flex' }}
+              placeholder="Must Select One"
+              field={withNameSpace('')}
+              options={classifications}
+            />
+            {toolTip && (
+              <div style={{ marginTop: '8px', marginLeft: '20px' }}>
+                <TooltipIcon toolTip={toolTip} toolTipId="classificationToolTip" />
+              </div>
+            )}
+          </Row>
+          <Row>
+            <FieldDescription>
+              A disposal project must be initiated in order to mark a property as "surplus active".
```

Comment:
This was subject to change if I remember correctly, but as mentioned above a lot of placeholder text right now.

Reply:
I don't think this is valid.  Technically a user can create a disposal project, add properties, change their classification and then remove them...  So I think it probably just makes sense to let them change the status here.  Additionally Sanjay was very clear it technically isn't up to SRES whether property is Surplus Active or Encumbered.